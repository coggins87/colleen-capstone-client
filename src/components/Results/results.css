.results_list {
  list-style-type: none;
}

.results {
  display: inherit;
  text-align: center;
  padding-bottom: 10%;
}
