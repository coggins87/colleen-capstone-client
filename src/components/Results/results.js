import React from 'react'

class Result extends React.Component{

  render (){
   
    return (<>
      <ul>
     <h3>In {this.props.time ? this.props.time : ''} minutes complete:</h3>
      {this.props.result.length > 0 ? this.props.result.map(movement=> {
        return (<li key={movement.id}>
          {movement.reps} {movement.movement_name} {movement.equipment ? `with ${movement.equipment}` : ''}
          </li>)
      }): '' }
    </ul>
    </>
    );
     
  }
}

export default Result