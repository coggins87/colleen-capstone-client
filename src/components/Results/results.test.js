import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Results from "./results";

jest.mock("uuid", () => jest.fn(() => "1"));

describe(`results component`, () => {
  const props = {
    testObj: {},
    testItem: "test Item",
    testCheck: () => {},
    result: [{ test: "q" }, { test: "g" }, { test: [{}, {}] }]
  };

  it("renders the results given props", () => {
    const wrapper = shallow(<Results {...props} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
