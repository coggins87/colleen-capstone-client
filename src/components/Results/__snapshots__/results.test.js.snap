// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`results component renders the results given props 1`] = `
<div
  className="results"
>
  <ul
    className="results_list"
  >
    <h3>
      In 
       minutes complete:
    </h3>
    <li
      key="1"
    >
       
       
    </li>
    <li
      key="1"
    >
       
       
    </li>
    <li
      key="1"
    >
       
       
    </li>
  </ul>
</div>
`;
